Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -184,6 +184,10 @@ further storage locations configured outside of :file:`fileadmin/`, even
 pointing to different servers or using 3rd party digital asset management
 systems.
 
+Depending on the configuration in
+:ref:`$GLOBALS['TYPO3_CONF_VARS']['BE']['fileadminDir'] <typo3ConfVars_be_fileadminDir>`
+another folder name than :file:`fileadmin/` can be in use.
```

Comment:
I think we should also warn that not all extensions might respect this setting?

Reply:
This should then be done in the section of the configuration setting, which is not in the scope of this PR. The note only hints, that a fileadmin folder might not be available.

Reply:
And: If you change it you have to take care about renaming the existing fileadmin folder and change the storage record accordingly.

Reply:
Ack